cli/train.go

```diff
@@ -23,6 +24,31 @@ const (
 	trainingStatusPrefix = "TRAINING_STATUS_"
 )
 
+// DataSubmitCustomTrainingJob is the corresponding action for 'data train submit-custom'.
+func DataSubmitCustomTrainingJob(c *cli.Context) error {
```

Comment:
Sorry if this was originally implemented by me, but I think this and below should be MLSubmit and the action is just `viam train`

Reply:
Oh, right! You are 100% correct 🤦